src/components/AllArticles.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from "react";
+import { fetchArticles } from "../utils/api-fetch-articles";
+import { Link } from "react-router-dom";
+
+const AllArticles = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    fetchArticles().then((articlesData) => setArticles(articlesData));
+  }, []);
+
+  return (
+    <div>
+      <h2>All Articles</h2>
+      <Link to="/">
+        <button>Home</button>
+      </Link>
+      <ul>
+        {articles.map((article) => (
+          <li className="single-article" key={article.id}>
+            {article.title}
```

Comment:
looks good so far but at this point i'd like to see a little more info in each article, such as the img, no of votes, date created in a human readable format, and comment count too. See this resource for info on how to add this in a concise way: https://www.refactoringui.com/previews/labels-are-a-last-resort